src/App.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Missions from './components/Missions';
+import Profiles from './components/Profiles';
+import Rockets from './components/Rockets';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <Routes>
+        <Route path="/" element={<Rockets />} />
+        <Route path="/missions" element={<Missions />} />
+        <Route path="/my-profile" element={<Profiles />} />
+      </Routes>
+    </div>
+  );
+}
```

Comment:
- [ ] When I navigate through the rockets and missions page, the rockets I reserved and missions I joined are no longer marked as reserved nor joined. As mentioned in the instructions, 'Make sure you only dispatch those actions once and do not add data to store on every re-render (i.e. when changing views / using navigation)'. You can fix it by dispatching your actions in a central component like this one, 'App.js' so that the actions are dispatched once 👌